src/common/BasedBreadCrumb/BasedBreadCrumb.tsx

```diff
@@ -34,8 +34,8 @@ export const BasedBreadCrumb = ({ breadcrumbs }: Props) => {
       <Breadcrumb>
         <BreadcrumbList data-testid="breadCrumb">
           {updatedBreadCrumbs.map((breadcrumb, index) => (
-            <BreadcrumbItem key={index}>
-              <>
+            <div key={`breadCrumb-${index}`} className="flex items-center">
```

Comment:
Index as key is a wrong pattern.

Reply:
Shouldn't ul be the parent of li?

Reply:
Sir @Monfernape,  Here is the `ol` element, which serves as the parent by default in ShadCN breadcrumbs.